spec.bs

```diff
@@ -496,7 +496,10 @@ A <dfn>component</dfn> is a [=struct=] with the following [=struct/items=]:
   1. Let |search| be the empty string.
   1. Let |hash| be the empty string.
   1. Let |inputs| be an empty [=list=].
-  1. [=list/Append=] |input| to |inputs|.
+  1. If |input| is a {{URL}}:
+    1. [=list/Append=] the [=URL serializer|serialization=] of |input| to |inputs|.
+  1. Otherwise:
+    1. [=list/Append=] |input| to |inputs|.
```

Comment:
```suggestion
  1. If |input| is a [=/URL=], then [=list/append=] the [=URL serializer|serialization=] of |input| to |inputs|.
  1. Otherwise, [=list/append=] |input| to |inputs|.
```

Reply:
It's not a URL object and we don't need this many substeps I think.

Reply:
Thanks, applied!